grails-app/controllers/org/pih/warehouse/api/InventoryTransactionSummaryApiController.groovy

```diff
@@ -0,0 +1,21 @@
+package org.pih.warehouse.api
+
+import grails.converters.JSON
+import grails.validation.ValidationException
+import org.pih.warehouse.inventory.InventoryTransactionSummaryService
+import org.pih.warehouse.inventory.InventoryTransactionsSummary
+import org.pih.warehouse.report.CycleCountReportCommand
+
+class InventoryTransactionSummaryApiController {
+
+    InventoryTransactionSummaryService inventoryTransactionSummaryService
+
+    def getInventoryTransactionsSummary(CycleCountReportCommand command) {
```

Comment:
can you rename CycleCountReportCommand to something else now that it's not just cycle count related?